ui-testing-core/src/main/java/com/avito/android/test/matcher/CompoundDrawableMatcher.kt

```diff
@@ -30,17 +30,17 @@ class CompoundDrawableMatcher(
         this.description = getDescription(context.resources)
 
         return left.matchDrawable(context, compoundDrawables[LEFT], tint) &&
-                right.matchDrawable(context, compoundDrawables[RIGHT], tint) &&
-                top.matchDrawable(context, compoundDrawables[TOP], tint) &&
-                bottom.matchDrawable(context, compoundDrawables[BOTTOM], tint)
+            right.matchDrawable(context, compoundDrawables[RIGHT], tint) &&
+            top.matchDrawable(context, compoundDrawables[TOP], tint) &&
+            bottom.matchDrawable(context, compoundDrawables[BOTTOM], tint)
     }
 
     private fun getDescription(resources: Resources): String {
         return "left=${left.getResourceName(resources)}, " +
-                "top=${top.getResourceName(resources)}, " +
-                "right=${right.getResourceName(resources)}, " +
-                "bottom=${bottom.getResourceName(resources)}," +
-                "with tint=${tint?.toString(16) ?: "not applied"}"
+            "top=${top.getResourceName(resources)}, " +
```

Comment:
is this a `kotlin official codestyle` settings? i bet not, because i already applied it on whole project.
Also PR bloated with these changes, that makes it hard to review. Better to do it in separate one

Reply:
I've avoided most of that changes by re applying project's code styles